docs/chapter_4/lesson_12.md

```diff
@@ -0,0 +1,77 @@
+Goal:
+* Learners should understand more information about 3 modules in app_state(auth, bank, staking)
+
```

Comment:
Learners should be able to add new account to auth, add initial balances to bank